File: fastFramework/src/main/java/com/tesla/framework/component/imageloader/IImagePahtFromCache.java
package com.tesla.framework.component.imageloader;

import android.graphics.Bitmap;

/**
 * Created by 01370340 on 2017/11/21.
 */

public interface IImagePahtFromCache {

    String getImagePahtFromCache(String url);

    Bitmap getBitmapFromCache(String url);
}

File: fastFramework/src/main/java/com/tesla/framework/component/imageloader/IImagePahtFromCache.kt
package com.tesla.framework.component.imageloader

import android.graphics.Bitmap

/**
 * Created by 01370340 on 2017/11/21.
 */
interface IImagePahtFromCache {

    fun getImagePahtFromCache(url: String?): String?

    fun getBitmapFromCache(url: String?): Bitmap?
}